StRoot/StarGenerator/Pythia8_2_35/StarPythia8.cxx

```diff
@@ -30,7 +30,7 @@ StarPythia8::StarPythia8(const char *name) : StarGenerator(name)
   // auxilliary files required by pythia. 
   // NOTE:  When adding new versions of Pythia8, we need to specify
   // the version of the code in the source code
-  TString path = "StRoot/StarGenerator/"; path+= Pythia8_version; path+="/xmldoc/"; 
+  TString path = "StRoot/StarGenerator/"; path+= Pythia8_version; path+="/share/xmldoc/"; 
```

Comment:
Wasn't this fixed in #3 already? What am I missing?

Reply:
I merged the main branch into your PR branch and this change does not show up anymore. An easy fix.